components/search/NotFound/index.tsx

```diff
@@ -1,29 +1,33 @@
-import NotFoundImage from 'public/img/notFound.svg';
+import NotFoundImage from 'public/img/notFound.png';
 import Image from 'next/image';
 import { PropsType } from './NotFound.type';
 import Link from 'next/link';
 
 function NotFound(props: PropsType) {
   const { query } = props;
   return (
-    <div className="container px-25 md:px-0 py-30 md:py-[120px] flex flex-col items-center">
-      <Image
-        src={NotFoundImage}
-        className="h-[148px] md:h-[318px]"
-        alt="Not Found"
-      />
-      <div className="text-16  md:text-45 pt-[45px] md:pt-[100px] text-center text-primary-500 font-semibold">
-        Oops! We couldn't find what you were looking for.
+    <div className="px-25 md:px-0 py-[75px] md:py-[120px] flex flex-col items-center max-w-3xl">
```

Comment:
please use the `container` class instead of using max-w manually.

```suggestion
    <div className="px-25 md:px-0 py-[75px] md:py-[120px] flex flex-col items-center container">
```

Reply:
Th `container` class applies `width: 1180px` for large screens but in Figma its `width` is equal to `740px` so I can not use that here.